mkdocs.yml

```diff
@@ -37,7 +37,7 @@ nav:
   - 'Examples': 'examples.md'
   - 'Clients': 'clients.md'
   - 'Design':
-    - 'Sub-Interpreters': 'subinterpreters.md'
+    - 'Isolation Technics': 'isolation_technics.md'
```

Comment:
@MeirShpilraien also needs to be renamed in runtime.md

Reply:
:+1: